just a temporary (pretty much useless) workaround that allows to start the GUI with wxPython 2.9.4
--- gui/wxpython/core/globalvar.py.orig
+++ gui/wxpython/core/globalvar.py
@@ -59,7 +59,7 @@ def CheckForWx():
             raise ValueError('Your wxPython version is %s.%s.%s.%s' % tuple(version.split('.')))
         if map(int, version.split('.')) >= unsupportedVersion:
             print >> sys.stderr, 'ERROR: wxGUI does not support wxPython %s yet.' % version
-            sys.exit(1)
+            # sys.exit(1)
     except ImportError, e:
         print >> sys.stderr, 'ERROR: wxGUI requires wxPython. %s' % str(e)
         sys.exit(1)
